groups-domains.php

```diff
@@ -88,6 +88,11 @@
                         </div>
                     </div>
                 </div>
+                <div>
+                    <p><strong>Note:</strong><br>
+                    The domain or regex filter will be assigned to the Default Group only.<br>
+                    Other groups can optionally be assigned within <a href="groups-domains.php">Group Management > Domains</a>.</p>
```

Comment:
The link appears at the moment not only on "Blacklist" and "Whitelist" but also at Group Management > Domains which is a bit pointless. Adding something like below changes the text/link depending on the loaded site.


```suggestion
                    Other groups can optionally be assigned
                    <?php if ($type == "white" || $type == "black") {
                      echo ' within <a href="groups-domains.php">Group Management > Domains</a>.</p>';
                    }
                    else {
                      echo 'in the table below.';
                    } ?>
```

Reply:
Or something like this:

```html
                <div>
                  <p><strong>Note:</strong><br>
                    The domain or regex filter will be automatically assigned to the Default Group.<br>
                    Other groups can optionally be assigned
                    <?php if ($type != "all") { ?>
                    within <a href="groups-domains.php">Group Management > Domains</a> page.
                    <?php } else {?>
                    in the list below <i>(using <b>Group assignment</b> column)</i>.
                    <?php } ?>
                  </p>
                </div>
```

... your choice.

Reply:
Go for the second one ;-)

Reply:
Good catch...  and well, I went for a mixture.

I had a look in the overall file and it seems we are using a structure like `if === white / black` in two other places. So I chose the `if ($type == "white" || $type == "black") ` to be explicit if someone else reads this and someone wants to change the structure in the future.
But instead of the echo, I used the ?> in line with the button decision "Add to Whitelist" vs "Add to Blacklist".

I also noticed that in the other instances === instead of == is used. Hence I used `if ($type === "white" || $type === "black")`.

Reply:
I also have to admit that I didn't go for the italics for the comment in brackets. It looked a little bit too busy for me and italics don't seem to be used much in the overall repo. But do let me know if you feel passionate about it.

Reply:
As I said... It's your choice.